packages/babel-parser/typings/babel-parser.d.ts

```diff
@@ -180,3 +180,12 @@ export const tokTypes: {
   // todo(flow->ts) real token type
   [name: string]: any;
 };
+
+export interface ParseError {
+  code: string;
+  reasonCode: string;
+}
+
+type ParseResult<Result> = Result & {
+  errors: ParseError[];
```

Comment:
This isn't always defined (only if errorRecovery is enabled)
```suggestion
  errors?: ParseError[];
```

Reply:
`parse` and `parseExpression` always returns `errors`(When `errorRecovery` isn't enabled it is empty array).

```js
const { parse } = require("@babel/parser");

const ast = parse("const foo = 'foo';");

console.log(ast.errors); // []
```